packages/gatsby-plugin-netlify-cms/src/cms.js

```diff
@@ -1,5 +1,27 @@
 import CMS from "netlify-cms"
 
+// The following window and global config settings below were taken from here.
```

Comment:
to me this is a different PR. If I read correctly this will fix support for storybook and has nothing to do with import compnents in cms.js? Am I correct?

Reply:
This is also relevant here because netlify runs it's own webpack instance, and encounters the same build issues as storybook's webpack instance when it encounters gatsby-related components, i.e., <Link>, <StaticQuery>, etc...